pkg/policy/api/rule_validation.go

```diff
@@ -219,6 +235,14 @@ func (e *EgressRule) sanitize() error {
 		}
 	}
 
+	if len(e.ICMPs) > 0 && !(viper.GetBool(option.EnableICMPRule)) {
+		return fmt.Errorf("ICMP rules can only be applied when the \"%s\" flag is set", option.EnableICMPRule)
+	}
```

Comment:
```suggestion
	if len(e.ICMPs) > 0 && !option.Config.EnableICMPRule {
		return fmt.Errorf("ICMP rules can only be applied when the %q flag is set", option.EnableICMPRule)
	}
```

Reply:
We don't need to import viper to do this check, all the options are loaded in `option.Config`.